travis_tools.sh

```diff
@@ -8,15 +8,44 @@ TERRYFY_DIR=$(dirname "${BASH_SOURCE[0]}")
 (cd $TERRYFY_DIR && git submodule update --init)
 source $TERRYFY_DIR/multibuild/osx_utils.sh
 
-MACPORTS_URL=https://distfiles.macports.org/MacPorts
-MACPORTS_PKG=MacPorts-2.3.3-10.9-Mavericks.pkg
+MACPORTS_VERSION=2.3.5
+MACPORTS_URL=https://github.com/macports/macports-base/releases/download/v$MACPORTS_VERSION
 MACPORTS_PREFIX=/opt/local
 MACPORTS_PY_PREFIX=$MACPORTS_PREFIX$MACPYTHON_PY_PREFIX
 # -q to avoid this:
 # https://lists.macosforge.org/pipermail/macports-users/2014-June/035672.html
 PORT_INSTALL="sudo port -q install"
 NIPY_WHEELHOUSE=https://nipy.bic.berkeley.edu/scipy_installers
 
+# Work round bug in travis xcode image described at
+# https://github.com/direnv/direnv/issues/210
+shell_session_update() { :; }
```

Comment:
Would you mind to move this "workaround" before line 8?
The braces in line 8 create a subshell which then exits and tries to call `shell_session_update`.
If the calling bash script has already called `set -e` the whole script is aborted.
(If my assumptions are correct).
See one failed travis run [here](https://travis-ci.org/tsparber/pygame/jobs/176945706#L597).

I hope `rvm` gets updated soon 😉 .